doc/adr/0004-molgenis-core-apps.md

```diff
@@ -0,0 +1,88 @@
+# 4. Molgenis Core Apps
+
+Date: 2020-10-22
+
+## Status
+
+Proposal
+
+## Context
+
+See [frontend architecture sessions](https://docs.google.com/document/d/1VW3ah5VAvAz2KnqNZlNmVqCzFhBMlIcjPPUlsHMFRIY/)
+for additional background information on micro-frontend/SPA architecture.
+
+Molgenis is flexible software that allows users to create their own apps.
+This is quite powerful whenever a new, custom application needs to be made
+on top of the Molgenis backend. However, the same modular paradigm is also
+being applied to parts of the frontend that are always being shipped with
+Molgenis. We call these admin-like apps Molgenis "core" packages.
+
+As an assessment; there are at the moment approximately 11 legacy Molgenis apps
+that are still rooted in Molgenis freemarker templates and Java controllers:
+
+* feedback
```

Comment:
This data won't age well. Maybe you should describe it as:

Whenever we make a new application for a feature and we are going to use that in production, we will always look to consolidate this app within exisiting apps if any other apps already have similarities. Either by user role (like admin tools) or kinship

Reply:
Sounds good.